erc7824-docs/docs/guides/migration-guide.md

```diff
@@ -12,6 +12,419 @@ import TabItem from '@theme/TabItem';
+        { "asset": "eth", "amount": "0.5" }
+      ],
+      "scope": "app.create",
+      "expires_at": 1719123456789
+    }, 1619123456789],
+    "sig": ["0x..."]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="root" label="Root Auth (Clearnode)">
+
+  ```json
+  {
+    "req": [1, "auth_request", {
+      "address": "0x1234567890abcdef...",
+      "session_key": "0x9876543210fedcba...",
+      "application": "clearnode",
+      "allowances": [],
+      "scope": "app.create",
+      "expires_at": 1750659456789
+    }, 1619123456789],
+    "sig": ["0x..."]
+  }
+  ```
+
+  </TabItem>
+</Tabs>
+
+#### API: Zero Deposit Channel Creation
+
+The `create_channel` method no longer accepts initial deposit parameters:
+
+<Tabs>
+  <TabItem value="before" label="Before">
+
+  ```json
+  {
+    "req": [1, "create_channel", {
+      "chain_id": 137,
+      "token": "0xeeee567890abcdef...",
+      "amount": "100000000",
+      "session_key": "0x1234567890abcdef..."
+    }, 1619123456789],
+    "sig": ["0x9876fedcba..."]
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="after" label="After">
+
+  ```json
+  {
+    "req": [1, "create_channel", {
+      "chain_id": 137,
+      "token": "0xeeee567890abcdef..."
+      // amount and session_key parameters removed
+    }, 1619123456789],
+    "sig": ["0x9876fedcba..."]
+  }
+  ```
+
+  </TabItem>
+</Tabs>
+
+#### API: Session Key Management Methods
+
+New methods for session key operations have been added.
+
+**Get Session Keys Request:**
+```json
+{
+  "req": [1, "get_session_keys", {}, 1619123456789],
+  "sig": ["0x..."]
+}
+```
+
+**Response includes session key details with application, allowances, and expiration:**
+```json
+{
+  "res": [1, "get_session_keys", {
+    "session_keys": [{
+      "id": "sk_123",
+      "session_key": "0x9876543210fedcba...",
+      "application": "My Trading App",
+      "allowances": [
+        { "asset": "usdc", "amount": "1000.0", "used": "250.0" }
+      ],
+      "scope": "app.create",
+      "expires_at": 1719123456789,
+      "created_at": 1619123456789
+    }]
+  }, 1619123456789],
+  "sig": ["0x..."]
+}
+```
+
+**Revoke Session Key Request:**
+```json
+{
+  "req": [1, "revoke_session_key", {
+    "session_key": "0x1234567890abcdef..."
+  }, 1619123456789],
+  "sig": ["0x..."]
+}
+```
+
+#### API: Operation Restrictions with Non-Zero Channel Balances
+
+The following operations will return errors if the user has any channel with non-zero amount:
+
+- **Transfer**: Returns error code indicating blocked due to non-zero channel balance
+- **Submit App State** (with deposit intent): Rejected if attempting to deposit
+- **Create App Session** (with allocations): Cannot create funded app sessions
+
+Error response example:
+```json
+{
+  "res": [1, "error", {
+    "error": "operation denied: non-zero allocation in 1 channel(s) detected owned by wallet 0x1234567890abcdef..."
+  }, 1619123456789],
+  "sig": ["0x..."]
+}
+```
+
+### Migration Summary
```

Comment:
I think we should change changes headings to the ones below, i.e. in Clearnode API the "#### Authentication Request with Enhanced Session Keys" to "#### Update Authentication Request Parameters". This way the whole guide reads like a list of steps right away, and we can remove the Migration summary.